scripts/release-train-issue-template.md

```diff
@@ -43,14 +43,14 @@ Wind down PR queue. There has to be enough time after the last (non-trivial) PR
 - [ ] Make sure there are no stray staging repos on sonatype
 - [ ] Wait until [master build finished](https://travis-ci.org/akka/akka-http/builds/) after merging the release notes
 - [ ] Create a [new release](https://github.com/akka/akka-http/releases/new) with the next tag version (e.g. `v13.3.7`), title and release description linking to announcement, release notes and milestone.
-- [ ] Check that the Travis CI release build has executed successfully
+- [ ] Check that the Travis CI [release build](https://travis-ci.org/akka/akka-http/branches) has executed successfully
 - [ ] Go to https://bintray.com/akka/maven/akka-http and select the just released version
-- [ ] Go to the Maven Central tab and sync with Sonatype
+- [ ] Go to the Maven Central tab and sync with Sonatype. You may need to log in and switch to the 'Old Look' for this.
 - [ ] Log in to Sonatype to Close the staging repository (optional, should happen automatically if selected in Bintray)
 - [ ] Notify Telemetry / Play team to check against staged artifacts
 - [ ] Run a test against the staging repository to make sure the release went well, for example by using https://github.com/akka/akka-http-quickstart-scala.g8 and adding the sonatype staging repo with `resolvers += "Staging Repo" at "https://oss.sonatype.org/content/repositories/comtypesafe-xxx"`
 - [ ] Release the staging repository to Maven Central.
-- [ ] Checkout the newly created tag and run `sbt -Dakka.genjavadoc.enabled=true ++2.12.8 "deployRsync akkarepo@gustav.akka.io"` to deploy API and reference documentation.
+- [ ] Checkout the newly created tag and run `sbt -Dakka.genjavadoc.enabled=true ++2.12.9 "deployRsync akkarepo@gustav.akka.io"` to deploy API and reference documentation.
```

Comment:
Why not 2.12.10?

Reply:
ah, no reason, just 'at least .9' ;). perhaps there are even scaladoc improvements in 2.13 actually

Reply:
ah yes https://github.com/scala/scala/pull/7834 is a good motivation to start moving towards that